_playwright-tests/UI/Templates.spec.ts

```diff
@@ -8,11 +8,29 @@ test.describe('Templates', () => {
   }) => {
     await navigateToTemplates(page);
     await closePopupsIfExist(page);
-
-    const AddButton = page.locator('[data-ouia-component-id="create_content_template"]');
-
-    // Wait for the Add button to become enabled (up to 10 seconds)
-    await AddButton.first().isEnabled({ timeout: 10000 });
+    await page.getByRole('button', { name: 'Add content template' }).click();
+    await page.getByRole('button', { name: 'Select architecture' }).click();
+    await page.getByRole('option', { name: 'aarch64' }).click();
+    await page.getByRole('button', { name: 'Select version' }).click();
+    await page.getByRole('option', { name: 'el9' }).click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Locate the checkbox in the table row and click it.
+    await page.locator('table tr:first-child input[type="checkbox"]').click();
```

Comment:
This will blindly select the first checkbox. Please make sure you select the correct repo. See how @swadeley solved it in https://github.com/content-services/content-sources-frontend/pull/495/files#diff-25bceba0e80c79787d7659137549d2f0f3d9190b4ed0db86c86572c51d65c08dR55

Reply:
Done!